lisa/tests/scheduler/load_tracking.py

```diff
@@ -975,7 +975,7 @@ def get_trace_cpu_util(self):
             # Ignore the first quarter of the util signal of each phase, since
             # it's impacted by the phase change, and util can be affected
             # (rtapp does some bookkeeping at the beginning of phases)
-            start += duration / 4
+            # start += duration / 4
```

Comment:
there were some issues (as per the comment) with the beginning, so it's quite likely that the problem now is just that we "cut" the signal in the middle of an activation, leading to having the 2nd half of an activation without its beginning. This would increase the average slightly.

If that's the problem, we can "cut" the signal at the first activation past the 4th of the duration. This way we will not have a half-activation.

Reply:
We ran the last 2 EAS mainline integrations with this patch. I haven't seen issues related to this patch in the other testcases which use this function. I kept the patch in the new version.

Reply:
>     Turns out that CPUMigrationBase.get_trace_cpu_util() advances 'start' of the window by 25% and CPUMigrationBase.get_expected_cpu_util() does not.

It looks like the other function is supposed to do but the code is also commented for unclear reasons, maybe by mistake. Have you tested removing the first 25% on both sides as well ?

Reply:
No I haven't. We could just go one way and see with the next integration.